packages/react-components/src/molecules/__tests__/ToastCard.test.tsx

```diff
@@ -12,11 +13,39 @@ it('does not render a toast when no message is provided', () => {
   const { queryByTitle } = render(<ToastCard>content</ToastCard>);
   expect(queryByTitle('Alert')).not.toBeInTheDocument();
 });
+it('renders a toast card with a link', () => {
```

Comment:
Just to test that any `ReactNode` is allowed? In that case doesn't seem necessary to me